DatabaseAndReporting/Integration/01_ManageQueries.md

```diff
@@ -0,0 +1,131 @@
+        The *Bind query parameters placeholders* window is displayed.
+        > [Info] There are two possible placeholders to define time conditions: **|datetime** and **|string**. The **|datetime** placeholder generates a calendar button in the *Bind query parameters placeholders* window, whereas the **|string** placeholder creates a free text field.
+
+        ![Bind query parameters placeholders](../../Assets/Screenshots/DatabaseAndReporting/ManagedQueries/Queries/BindQueryParamsPlaceholders.png "[Bind query parameters placeholders]")
+
+    - Define the desired time criteria in the *creationDate* field, either by entering the date with the keyboard or by clicking the calendar button to select the desired date and time. 
+
+    - Click the [EXECUTE] button.   
+        The query results are displayed in table format at the bottom of the workspace. 
+
+[comment]: <> (ETL use enabled muss noch ausgeblendet werden, laut JS)
+
+6. If desired, enter one or several IP addresses or an IP range (subnet mask) in the *IP whitelist (only public download)* field to allow access only to those IP addresses entered in case of public download.  
+    > [Info] When entering more than one IP address, each IP address must be entered on a separate line. Bear in mind that if no IP address is whitelisted, any person with the link, from any IP address, can access the query results via public download. If at least one IP address is whitelisted, all other IP addresses are blacklisted. 
+
+7. Select the *API and download enabled* checkbox to allow logged in users with the appropriate rights to execute the query and download the retrieved data.
+    > [Info] Additionally, user access rights must be granted for every single predefined query, see [Grant access rights to a user group](./05_ManageUserRights.md#grant-access-rights-to-a-user-group).
+
+8. Select the *Public download enabled* checkbox to allow any user, also non-logged in users, with the applicable link to execute the query and download the retrieved data via web browser. 
+    > [Info] A specific download link, containing a UUID (Universally Unique Identifier), is required to execute the query in a web browser.
+
+9. Select the *Write access enabled* checkbox to allow write operations when executing the query. Otherwise, a query with write operations, such as INSERT or DELETE, will not be executed and an error message will be displayed.
+
+10. If desired, click the [TRY QUERY (READ ONLY)] button to test the query.  
+    The query results are displayed in table format at the bottom of the workspace.
+
+11. Click the [SAVE] button.  
+   The query has been saved. The *Create MySQL query* is closed. The new query is displayed in the list of queries.
+
+
+## Edit a query
+
+After you have a created a query, it can be edited to change any previously set values. The fields *ID* and *UUID*, however, are automatically assigned by the system and cannot be modified.
```

Comment:
```suggestion
After you have a created a query, it can be edited to change any previously set values. The *ID* and *UUID* fields are automatically assigned by the system and cannot be modified.
```